pennylane_lightning/src/util/RuntimeInfo.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "RuntimeInfo.hpp"
+
+#include <array>
```

Comment:
You may want to move "include array" to line 50

Reply:
Changed to include `<array>` only for MSVC.